hbw/app/models/hbw/task.rb

```diff
@@ -36,22 +36,30 @@ def fetch(email, entity_code, entity_class, size = 1000, for_all_users = false)
         end
       end
 
-      def fetch_task_count(email, entity_code, entity_class, for_all_users = false)
+      def fetch_count(email)
         user = ::HBW::BPMUser.fetch(email)
+
+        unless user.nil?
+          do_request(:post,
+                     'task/count',
+                     assignee:   user.id,
+                     active:     true)['count']
+        end
+      end
+
+      def fetch_count_unassigned(email)
+        user = ::HBW::BPMUser.fetch(email)
+
         unless user.nil?
           do_request(:post,
                      'task/count',
-                     assignee:   assignee(user, email, for_all_users),
-                     active:     true,
-                     processVariables: [
-                       name:     HBW::Widget.config[:entities].fetch(entity_class)[:entity_code_key],
-                       operator: operation(entity_code),
-                       value:    entity_code
-                     ])['count']
+                     unassigned:    true,
```

Comment:
![image](https://user-images.githubusercontent.com/33573380/66133562-8efedf80-e5ff-11e9-922b-4f7f5f211373.png)

Reply:
Yes, I kept it just for explicity sake, but if default behaviour is well documented I think we can remove it.